bundle/tests/quality_monitor/databricks.yml

```diff
@@ -24,14 +31,14 @@ targets:
       quality_monitors:
         my_monitor:
           table_name: "main.test.staging"
-          output_schema_name: "staging"
+          output_schema_name: "main.test"
 
   production:
     resources:
       quality_monitors:
         my_monitor:
```

Comment:
what does this name mean?

Reply:
The original name wasn't valid according the the API. It requires something of the format `catalog.schema`. Actually, that seems like something to reflect on https://docs.databricks.com/api/workspace/qualitymonitors/create?